common-lib/src/main/scala/com/gu/mediaservice/lib/management/Management.scala

```diff
@@ -63,4 +66,20 @@ class ElasticSearchHealthCheck(override val controllerComponents: ControllerComp
       }
     }
   }
+
+  def imageCounts: Action[AnyContent] = Action.async {
+    elasticsearch.countImages().map {
+      case _ @(catCount: CatCountResponse, searchResponse: SearchResponse, indexStats: IndexStatsResponse) =>
+        val json: String =
+          s"""{
+             |  "catCount": ${catCount.count},
```

Comment:
> > @akash1810 what did you have in mind for the logging? Not sure what you mean by strong structure, but keen to get some better logging in here for sure.
> 
> If we log a object, then we will create distinct fields in ELK. For example:
> 
> ```js
> const logDetail = {
>   count: 100
> };
> 
> console.log("retrieved count", logDetail)
> ```
> 
> This will create a `count` field in ELK which we can use to graph on.

This has now been added, take a look at the lambda function on line 48.